index.html

```diff
@@ -5,13 +5,24 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>JavaScript Capstone project</title>
+    <link rel="stylesheet" href="style.css">
     <script src="https://kit.fontawesome.com/7415ea4535.js" crossorigin="anonymous"></script>
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200;300;400;500;600&display=swap" rel="stylesheet">
 </head>
 <body>
 
+    <div class="container">
+        <nav class="header">
+            <div class="logo"> <img src="./images/pokemonlogo.png" alt="Pokemon Gotta catch 'em all! "> </div>
+            <ul>
+                <li class="nav-item"> <a href="">Pokemon( 6 )</a> </li>
+                <li class="nav-item"> <a href=""> Moves </a> </li>
+                <li class="nav-item"> <a href=""> Abilities </a> </li>
+            </ul>
+        </nav>
+    </div>
```

Comment:
[OPTIONAL] It will be better to wrap this block in a `header` tag to align with good semantic rules

Reply:
Thanks for that highlight, I will implement that change right away